lua/nvim-drawer/init.lua

```diff
@@ -524,12 +687,6 @@ function mod.setup(_)
       for _, instance in ipairs(instances) do
         if instance.state.is_open then
           instance.open({ focus = false })
-
-          local winid = instance.get_winid()
-          instance.set_size(instance.state.size)
-
-          local bufnr = instance.state.previous_bufnr
-          vim.api.nvim_win_set_buf(winid, bufnr)
```

Comment:
Personal milestone!

It always irked me that the restore logic here was so involved, even going back to vim-drawer: https://github.com/mikew/vim-drawer/blob/38c760debb0e85a3586878ea40ff6cc978fea054/autoload/drawer.vim#L290-L300

Really, `.open` and `.close` should suss themselves out correctly without having to account for any weird side effects, and finally they do!